### ISSUE TITLE
Clarify MSRV policy

### ISSUE THREAD

**Issue by osiewicz** [pyrossh/rust-embed#216]
I'd like to make a contribution to reduce size of LLVM IR generated by this library, though it's unclear for me what's the MSRV of this library. It would be great to have that clarified in README.

**Comment by pyrossh** [comment]
I was thinking MSRV would be based on the latest edition supported. Right now the project is using Rust 2018 edition. We might need to upgrade it to 2021. Then I think MSRV would be Rust v1.56.0.

**Comment by osiewicz** [comment]
1.56 does not work for rust-embed itself (there are dependency errors for whatever reason) and while 1.60 works for just rust-embed, it fails with `compression` feature enabled. Seems like MSRV of `include-flate` is 1.63 at the very least. 

Either way, the linked PR does not break MSRV. Thanks!

**Comment by pyrossh** [comment]
Ahh it doesn't weird why we still specify edition in the toml. Thats a nice idea. We can get the latest MSRV based of the deps.

**Comment by osiewicz** [comment]
There's `cargo-msrv` we could use for that I guess

**Comment by pyrossh** [comment]
Tested it. It seems MSRV is `v1.60.0`.

**Comment by osiewicz** [comment]
Cool. That's just for `rust-embed` without `compression` feature though, right?

**Comment by pyrossh** [comment]
Yeah I think so. Couldn't find a way to run it with all the features enabled. So my assumption is thats what it does.
